cves/CVE-2016-3090.yml

```diff
@@ -103,9 +126,14 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    The original code was unit tested, however there were several test cases missing
+    from the unit tests that could be exploited.  The fix also updated these unit tests
```

Comment:
Got any examples of a missing test case that could've caught this issue?

Reply:
Unfortunately, I could not find anything, they were really vague about what actually caused it, this article is the closest thing, but it doesn't say anything about this vulnerability specifically which seems to be related to ANTLR.
https://pentest-tools.com/blog/exploiting-ognl-injection-in-apache-struts/